src/properties.json

```diff
@@ -1,5 +1,7 @@
 {
   "realm": "ClientFhirServer",
   "client": "app-login",
-  "auth": "http://localhost:8180/auth"
+  "auth": "http://localhost:8180/auth",
+  "server": "http://localhost:8090",
```

Comment:
Minor nit, but do we ever use this in the application?

Reply:
not right now but it is likely that in the future when we want to support something that isn't order review we will need to build the url to fetch like so: `config.server + state.version + state.service` instead of the unceremonious `config.cds_service_endpoint`.